tests/src/NetworkRequest.cpp

```diff
@@ -3,19 +3,24 @@
 #include "common/NetworkManager.hpp"
 #include "common/NetworkResult.hpp"
 #include "common/Outcome.hpp"
-#include "common/QLogging.hpp"
-#include "providers/twitch/api/Helix.hpp"
 
 #include <gtest/gtest.h>
+// Using this over <QTest> to avoid depending on the Test component of Qt
+// We're only interested in QTest::qWaitFor.
+#include <QtCore/qtestsupport_core.h>
 
 using namespace chatterino;
 
 namespace {
 
 #ifdef CHATTERINO_TEST_USE_PUBLIC_HTTPBIN
-const char *const HTTPBIN_BASE_URL = "http://httpbin.org";
+// Not using httpbin.org, since it can be really slow and cause timeouts.
+// postman-echo has the same API. They only differ for the 402 response.
+const char *const HTTPBIN_BASE_URL = "https://postman-echo.com";
+constexpr const bool CUSTOM_402_RESPONSE = false;
```

Comment:
We can remove our test for the 402 code instead, and that should let us only replace the HTTPBIN base url